src/handle_action.cpp

```diff
@@ -2082,6 +2041,36 @@ bool game::handle_action()
             default:
                 break;
         }
+        if( dx != 0 || dy != 0 ) {
+            if( !u.get_value( "remote_controlling" ).empty() &&
+                ( u.has_active_item( "radiocontrol" ) || u.has_active_bionic( bio_remote ) ) ) {
+                rcdrive( dx, dy );
+            } else if( veh_ctrl ) {
+                pldrive( dx, dy );
+            } else {
+                if( auto_travel_mode/* && safe_mode == SAFE_MODE_ON*/ ) {
```

Comment:
```suggestion
                if( auto_travel_mode ) {
```

Reply:
Thanks.

After a few tests I am considering making auto travel mode possible only when safe mode is enabled. I will test some more and either remove commented code or uncomment it.